controllers/argocd/ingress.go

```diff
@@ -18,16 +18,17 @@ import (
 	"context"
 	"fmt"
 
-	extv1beta1 "k8s.io/api/extensions/v1beta1"
+	networkingv1 "k8s.io/api/networking/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/util/intstr"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 
 	argoprojv1a1 "github.com/argoproj-labs/argocd-operator/api/v1alpha1"
 	"github.com/argoproj-labs/argocd-operator/common"
 	"github.com/argoproj-labs/argocd-operator/controllers/argoutil"
 )
 
+var PathTypeImplementationSpecific = networkingv1.PathTypeImplementationSpecific
```

Comment:
Can you use this networkingv1.PathTypeImplementationSpecific const directly instead of assigning to a var?

Reply:
Since it's a constant the compiler throws an error when I try to reference it directly because go doesn't allow accessing the address of a constsnt, so I had workaround it